tests/test_state/test_remote_state_manager.py

```diff
@@ -202,7 +232,10 @@ def test_provision_remote_state(
         state_storage_destination_path, state_storage_expected_tf_vars
     )
 
-    assert_matcha_config(matcha_testing_directory, expected_matcha_config)
+    print(matcha_testing_directory, valid_config_testing_directory)
```

Comment:
I guess this was for debugging, can be removed now?

Reply:
Good spot, sorted. Ty.